Impossible-Agent-Suite/non-whitelist-mint/src/agent.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Finding,
+  FindingType,
+  FindingSeverity,
+  TransactionEvent,
+  HandleTransaction,
+} from 'forta-agent';
+
+import {
+  Verifier,
+  IF_WHITELIST_VERIFIER,
+  IDIA_WHITELIST_VERIFIER,
+} from './verifier';
+
+// Address of the Impossible Finance token contract
+let IF_ADDR = '0xb0e1fc65c1a741b4662b813eb787d369b8614af1';
+let IDIA_ADDR = '0x0b15ddf19d47e6a86a56148fb4afffc6929bcb89';
+let ZERO_ADDR = '0x0000000000000000000000000000000000000000';
+
+// Array containing a token address and its respective whitelist verifier function
+const TOKENS: [string, Verifier, string, string][] = [
+  [
+    IF_ADDR,
+    IF_WHITELIST_VERIFIER,
+    'IF',
+    'IMPOSSIBLE-2-1'
+  ],
+  [
+    IDIA_ADDR,
+    IDIA_WHITELIST_VERIFIER,
+    'IDIA',
+    'IMPOSSBILE-2-2'
+  ]
+];
+
+// Impossible Finance token contract ABI with relevant events/functions
+export const IF_ABI= [
+  'event Transfer(address indexed from, address indexed to, uint256 value)',
+  'function staxMigrate(uint)'
```

Comment:
recheck function signature

Reply:
Argument name has been added to the `staxMigrate` function signature.